### PR TITLE
Adding Pavel's tool for generating MVA payloads

### PR DESCRIPTION
**Pull request by andrewj314**
This is a tool that Pavel implemented before he left that generates SQlite .db file for tau MVA payloads. We will update this shortly with a pared-down list of discriminators in order to reduce our overall footprint in CondDB

### CODE REVIEW

**Comment by slava77** [comment]
binary root files should go to an external repository, not into the cmssw source tree
[what's the size of these files, btw?]


https://github.com/cms-data?utf8=%E2%9C%93&query=Reco
please contact @degano or @smuzaffar to setup a repository and populate it

**Comment by slava77** [comment]
we will need PRs in 75X and 76X.
I suggest you submit a PR in 76X, we sort out all issues there
and then the backport can be made to 74X and 75X.

**Comment by andrewj314** [comment]
Ah okay, sure thing. The root files are ~10K, except for one which is ~200K

File: RecoTauTag/TauTagTools/plugins/TFormulaWriter.cc [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include "RecoTauTag/TauTagTools/plugins/TFormulaWriter.h"
```
Comment (slava77) [inline RecoTauTag/TauTagTools/plugins/TFormulaWriter.cc]: please move all non-production code to /test directory.
I don't see anything tau reco specific here.
This code should be in CondFormats/PhysicsToolsObjects/test or in a similar package.

File: RecoTauTag/TauTagTools/plugins/TGraphWriter.cc [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#include "RecoTauTag/TauTagTools/plugins/TGraphWriter.h"
```
Comment (slava77) [inline RecoTauTag/TauTagTools/plugins/TGraphWriter.cc]: same about TGraph: please move to CondFormats/PhysicsToolsObjects/test or a similar appropriate place in the Cond* sybsystem

**Comment by slava77** [comment]
about the /data files: why are they needed here in the release at all?
The only consumer now is the writer that creates a payload in the database.
Supposedly any production code will read directly from the database payload.
If it's the case, the root files are temporary and do not belong in the release at all.

**Comment by slava77** [comment]
-1 
based on the code review

**Pull request closed without merging**